views/create-event.html

```diff
@@ -16,7 +16,116 @@
 	<form id="create-event" method="POST">
     <label for="title">Event title:</label>
     <input type="text" name="title">
+    
+    <label for="location">Event location:</label>
+    <input type="text" name="location">
+    
+    <label for="image">Event image:</label>
+    <input type="text" name="image">
+    
+    <label for="year">Event year:</label>
+    <input type="text" name="year">
```

Comment:
The specification states that "The year, month, day, hour, and minute form elements should be select elements."

We do not need `input[type='text']` for each of them since a form should only have one input for each `name`.